api/app/controllers/mno_enterprise/jpi/v1/admin/tenants_controller.rb

```diff
@@ -11,12 +11,20 @@ def show
     # PATCH /mnoe/jpi/v1/admin/tenant
     def update
       @tenant = MnoEnterprise::Tenant.show
+      timestamp = Time.now.utc.to_i
+      tenant_params['frontend_config']&.merge!(config_timestamp: timestamp)
```

Comment:
If `tenant_params['frontend_config']` is `nil`, the timestamp is not updated so we'll wait forever for the cube to restart.

Reply:
What do you think of 
```ruby
tenant_params.deep_merge!({frontend_config: {config_timestamp: timestamp}}
```